testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jpa/secondlevelcache/JPA2LCTestCase.java

```diff
@@ -97,11 +96,7 @@
     public String getCacheRegionName() throws Exception {
 
         if (CACHE_REGION_NAME == null) {
-            SFSB2LC sfsb = lookup("SFSB2LC", SFSB2LC.class);
-            String prefix = sfsb.getCacheRegionName();
-
-            assertNotNull("'hibernate.cache.region_prefix' is null.", prefix);
-            CACHE_REGION_NAME = prefix + '.' + this.getClass().getPackage().getName() + '.';
+            CACHE_REGION_NAME = this.getClass().getPackage().getName() + '.';
```

Comment:
We could make `CACHE_REGION_NAME` final and statically assign the value.

Reply:
@scottmarlow Thanks; leaving it the way I had it would confuse future maintainers, wondering why a simple thing was done in a complicated way.

I made CACHE_REGION_NAME a constant and got rid of the getCacheRegionName method; callers just read the constant.